### PR TITLE
Describe content node thread pools used for feeding

### PR DESCRIPTION
**Pull request by geirst**
@toregge please review

@jobergum FYI

### CODE REVIEW

File: en/performance/sizing-feeding.html [hunk 1]
```diff
@@ -154,6 +150,152 @@ <h3 id="debugging-performance">Debugging performance</h3>
...
+  <tbody>
+
+  <tr>
+    <th rowspan="4">master</th>
+    <td colspan="2">
+      <p id="master-thread">
+      Updates the
+      <a href="../attributes.html#document-meta-store">document metastore</a>,
+      prepares tasks to the <a href="#index-thread">index</a> and <a href="#summary-thread">summary</a> threads,
+      and splits a write operation into a subset of tasks to update individual
```
Comment (toregge) [inline en/performance/sizing-feeding.html:180]: set instead of subset ?

File: en/performance/sizing-feeding.html [hunk 2]
```diff
@@ -154,6 +150,152 @@ <h3 id="debugging-performance">Debugging performance</h3>
...
+    <th style="white-space:nowrap;">Metric prefix</th>
+    <td><em>content.proton.documentdb.threading_service.master.</em></td>
+  </tr>
+
+  <tr>
+    <th rowspan="4">index</th>
+    <td colspan="2">
+      <p id="index-thread">
+      Manages writing of index fields in the <a href="../proton.html#index">memory index</a>.
+      It splits a write operation into a subset of tasks to update individual index fields,
```
Comment (toregge) [inline en/performance/sizing-feeding.html:205]: set instead of subset ?

**Review by toregge — APPROVED** [review]

File: en/performance/sizing-feeding.html (hunk 1 above)
Comment (geirst) [reply en/performance/sizing-feeding.html]: Fixed

File: en/performance/sizing-feeding.html (hunk 2 above)
Comment (geirst) [reply en/performance/sizing-feeding.html]: Fixed

**Pull request merged**